docs/get-started/v2/get-started/features/noise-cancellation.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: Noise Cancellation
+nav: 3.9
+---
+
+The Noise cancellation feature is an invaluable tool designed to enhance the audio quality in scenarios such as conferences, live streams, and recordings where unwanted background noise can degrade the listening experience. By leveraging advanced Artificial Intelligence (AI) algorithms, this feature intelligently identifies and suppresses extraneous sounds, ensuring clear and crisp audio output.
```

Comment:
either both N and C are capital or both are in small.

Reply:
Keep them in capital

Reply:
You can extend this to the entire doc

Reply:
Got it. Will make N and C caps only.